### ISSUE TITLE
Add right shift operator for int64

### ISSUE THREAD

**Issue by leafpetersen** [dart-lang/sdk#32838]
Non-breaking missing addition to int64:

"We will need front-end support to add `>>>` as an operator (probably even a constant operator), and I don't believe it is there yet."

**Comment by lrhn** [comment]
The `>>>` operator is already in defined in the specification.

**Comment by lrhn** [comment]
Duplicate of #30888 (and related) issues.

**Comment by lrhn** [comment]
This covers the actual use of the operator on the `int` class. It is blocked on #30886.
